### ISSUE TITLE
[FR] Display 'Start Time' as an absolute timestamp and not a relative reference

### ISSUE THREAD

**Issue by Nika-St** [mlflow/mlflow#7413]
### Willingness to contribute

No. I cannot contribute this feature at this time.

### Proposal Summary

In previous versions, 'Start Time' used to be displayed as an absolute timestamp (something like "%d.%m.%Y, %H:%M:%S"). At some point, it was changed to relative references (like '3 hours ago').
Please change this back or allow the user to choose. It is very inconvenient trying to identify the runs like this: first, you have to count the days, and second, the farther back, the less specific the references become: there can be hundreds of runs, say, 5 days ago - and they are all '5 days ago' and thousands of runs 5 months ago and they are all just '5 months ago'
Thanks!
<img width="219" alt="2022-11-28 (1)" src="https://user-images.githubusercontent.com/29167311/204302026-ec8c158e-64a1-4105-8ce1-5092f2684c2f.png">

### Motivation

> #### What is the use case for this feature?
Identifying runs

> #### Why is this use case valuable to support for MLflow users in general?
The point of using mlflow is to find previously logged experiment runs, also based on time

> #### Why is this use case valuable to support for your project(s) or organization?

> #### Why is it currently difficult to achieve this use case?
It is very inconvenient trying to identify the runs like this: first, you have to count the days, and second, the farther back, the less specific the references become: there can be hundreds of runs, say, 5 days ago - and they are all '5 days ago' and thousands of runs 5 months ago and they are all just '5 months ago'

### What component(s) does this bug affect?

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

### What interface(s) does this bug affect?

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

### What language(s) does this bug affect?

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

### What integration(s) does this bug affect?

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

**Comment by harupy** [comment]
@Nika-St Thanks for the FR. We'll discuss this in the team and get back to you.

**Comment by harupy** [comment]
@Nika-St You can view absolute timestamps by hovering on runs.

<img width="556" alt="image" src="https://user-images.githubusercontent.com/17039389/204420946-f5be3093-df92-49c8-98b2-b86e253440a7.png">

However, this is insufficient, right?

**Comment by Nika-St** [comment]
Thanks @harupy ! Yes? hovering over each run isn't convenient. Maybe it's possible to swap the two: show the timestamp permanently and 'X days/hours'etc. ago' on hover?

**Comment by harupy** [comment]
I don't think we can swap the two. We could add a toggle switch in the toolbar above the table or perhaps in the column header. Added the `help wanted` label so someone in the community can contribute.

**Comment by subramaniam02** [comment]
@harupy I am interested to work on this FR. I have tried something, attached some screenshots. Idea is based on having two columns, one relative and other absolute (not visible by default). Users will have the option to choose absolute date from `Columns` dropdown and optionally hide relative date column.

![Screenshot 2023-02-05 at 11 57 12 PM](https://user-images.githubusercontent.com/1637727/216916140-b0d57df4-081e-4503-aa74-cace41f2abfa.png)

![Screenshot 2023-02-05 at 11 57 20 PM](https://user-images.githubusercontent.com/1637727/216916003-d5f0bb47-331e-4908-ac41-13b503e55030.png)

Thanks !
